packages/web/src/app/Header.tsx

```diff
@@ -6,70 +6,70 @@ import DropdownMenu, {
 import { Icon } from '../shared/components/Icon';
 import apiClient from '../shared/apiClient';
 import useAuth from '../shared/hooks/useAuth';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 import { useAccessControl } from '../shared/accessControl';
 
+function navLinkClasses(props: { isActive: boolean; isPending: boolean }) {
```

Comment:
```suggestion
function navLinkClasses({ isActive }: { isActive: boolean }) {
```

No need to specify `isPending`, right?